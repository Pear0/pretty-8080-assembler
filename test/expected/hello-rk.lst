	                    ; üêü –¥–ª—è –†–∞–¥–∏–æ-86–†–ö
	                    .project hello.rk 
	                    .tape rk-bin
	                    .org 0
	                prompt  equ 0F86Ch
	                puts    equ 0F818h
	                
0000	21 09 00            lxi hl, msg
0003	CD 18 F8            call puts
0006	C3 6C F8            jmp prompt
	                
	                msg:
0009	1F 70 72 69         db 1fh,'priwet, mir!',0dh,0ah,0
	. . .
0015	21 0D 0A 00 

	                
	                
	                
Labels:
msg                     0009
prompt                  F86C	puts                    F818

Memory dump:
0000:  21 09 00 CD 18 F8 C3 6C-F8 1F 70 72 69 77 65 74  !......l..priwet
0010:  2C 20 6D 69 72 21 0D 0A-00                       ,.mir!..........
