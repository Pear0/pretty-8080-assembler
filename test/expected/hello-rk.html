<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><link href="list-test.css" rel="stylesheet" type="text/css" media="screen"/><link href="../list-test.css" rel="stylesheet" type="text/css" media="screen"/></head><body><pre id="l0"><span class="adr"></span>	                    <span class="cmt">; üêü –¥–ª—è –†–∞–¥–∏–æ-86–†–ö</span></pre>
<pre id="l1"><span class="adr"></span>	                    <span id="code1">.project hello.rk </span></pre>
<pre id="l2"><span class="adr"></span>	                    <span id="code2">.tape rk-bin</span></pre>
<pre id="l3"><span class="adr"></span>	                    <span id="code3">.org 0</span></pre>
<pre id="l4"><span class="adr"></span>	                <span class="l" id="label4">prompt</span>  <span id="code4">equ 0F86Ch</span></pre>
<pre id="l5"><span class="adr"></span>	                <span class="l" id="label5">puts</span>    <span id="code5">equ 0F818h</span></pre>
<pre id="l6"><span class="adr"></span>	                </pre>
<pre id="l7"><span class="adr">0000</span>	21 09 00            <span id="code7">lxi hl, msg</span></pre>
<pre id="l8"><span class="adr">0003</span>	CD 18 F8            <span id="code8">call puts</span></pre>
<pre id="l9"><span class="adr">0006</span>	C3 6C F8            <span id="code9">jmp prompt</span></pre>
<pre id="l10"><span class="adr"></span>	                </pre>
<pre id="l11"><span class="adr"></span>	                <span class="l" id="label11">msg</span><span id="code11">:</span></pre>
<pre id="l12"><span class="adr">0009</span>	1F 70 72 69         <span id="code12">db 1fh,'priwet, mir!',0dh,0ah,0</span><br/>	.&nbsp;.&nbsp;.&nbsp;<br/>0015	21 0D 0A 00 <br/></pre>
<pre id="l13"><span class="adr"></span>	                </pre>
<pre id="l14"><span class="adr"></span>	                </pre>
<pre id="l15"><span class="adr"></span>	                </pre>
<pre>Labels:</pre><div class="hordiv"></div><pre class="labeltable"><span class='t1' onclick="return gotoLabel('msg');">msg                     0009</span><span class='t1' onclick="return gotoLabel('prompt');">prompt                  F86C</span><span class='t1' onclick="return gotoLabel('puts');">puts                    F818</span></pre><div>&nbsp;</div><pre>Memory dump:</pre><div class="hordiv"></div><pre class="d0">0000:  21 09 00 CD 18 F8 C3 6C-F8 1F 70 72 69 77 65 74  !......l..priwet</pre><br/><pre class="d1">0010:  2C 20 6D 69 72 21 0D 0A-00                       ,.mir!..........</pre><br/></body></html>